source/delegate-factory/truffle-config.js

```diff
@@ -14,11 +18,23 @@ module.exports = {
       gas: 0xfffffffffff,
       gasPrice: 0x01,
     },
+    rinkeby: {
+      provider: () =>
+        new HDWalletProvider(
+          process.env.MNEMONIC,
+          'https://rinkeby.infura.io/v3/' + process.env.INFURA_API_KEY
```

Comment:
can we check, if someone doesn't have a .env file, with these 3 params, even if they're empty, will this block testing?

Reply:
it would only block using it on rinkeby.

Reply:
tested and seems fine.